web/app/Http/Controllers/PageController.php

```diff
@@ -28,45 +214,29 @@ public function show(): View
         $timestamp = null;
         $tags = null;
 
-        if ($values['site']) {
-            $title = $values['site']->getName();
+        $title = $site->getName();
+
+        // TODO: this doesn't seem elegant?
+        $sidebar_page = Page::findSlug($site_id, 'nav:side', false, true);
```

Comment:
It's not no, and right now we're hardcoding `nav:(top|side)` as the navigation pages. (Wikidot has them configurable, though honestly I'm not sure inherently how valuable being able to configure that is lol)

We will eventually want a utility that can pull navigation data for a site as a single encapsulated function. (Wikidot doesn't have that because it does a query every time for what the navigation pages are...)